### PR TITLE
20 docasauraus setup our plugin documentation

### PR DESCRIPTION
**Pull request by unicop**
# Motivation

Having pretty docs will help us gain users and show a certain level of the profession.

- I skipped the part of deployment for now because I wanted to open the PR and push it forward, I'm opening myself the following ticket to take care of deployment.

you can run the website by yourself using `npm run start` on the `website/` dir

### CODE REVIEW

File: website/src/pages/index.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
+import React from 'react';
```
Comment (neriyarden) [inline website/src/pages/index.tsx:1]: @unicop from what i understand, the homepage is more customizable so it supports react?
or can react be used in any page?

File: website/blog/2021-08-01-mdx-blog-post.mdx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+---
+
+Blog posts support [Docusaurus Markdown features](https://docusaurus.io/docs/markdown-features), such as [MDX](https://mdxjs.com/).
+
+:::tip
+
+Use the power of React to create interactive blog posts.
+
+```js
+<button onClick={() => alert('button clicked!')}>Click me!</button>
```
Comment (neriyarden) [inline website/blog/2021-08-01-mdx-blog-post.mdx:15]: Nice cross-over with react :)

**Review by neriyarden — APPROVED** [review]
Very cool 🤩
We will need a domain for the website right?

**Comment by unicop** [comment]
> Very cool 🤩 We will need a domain for the website, right?

we can use `github-pages` for now, it is an open-source, and we don't need to pay for a domain

File: website/src/pages/index.tsx (hunk 1 above)
Comment (unicop) [reply website/src/pages/index.tsx:1]: can be used anywhere! although as more as we maintain more code, it will cost us more time

Comment (unicop) [reply website/src/pages/index.tsx:1]: but generally speaking, most of the sites can be customized.

Comment (neriyarden) [reply website/src/pages/index.tsx:1]: cool 😎

**Pull request merged**